tests/wpt/tests/dom/events/scrolling/scroll-event-fired-to-element.html

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Scroll event should behave correctly for Element.offsetTop and Element.offsetLeft</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src="scroll_support.js"></script>
```

Comment:
Did you forget to `git add` this file?

Reply:
I think I have updated it. Please correct me if I am missing something.